CAFMaker/HeaderFiller.cxx

```diff
@@ -21,6 +22,27 @@ namespace caf
     stdrec.hdr.subrun = subrun;
     stdrec.hdr.evt    = spillNum;
 
+    art::Handle<rb::Spill> spillHandle;
+    try {
+      evt.getByLabel("spillinfo",spillHandle);
+      stdrec.hdr.timestamp = spillHandle->Timestamp();
+      stdrec.hdr.intensity = spillHandle->Intensity();
+      stdrec.hdr.momentum  = spillHandle->Momentum();
+      stdrec.hdr.gcp5      = spillHandle->MT5CPR();
+      stdrec.hdr.gcp6      = spillHandle->MT6CPR();
+      stdrec.hdr.gcp5iFix  = spillHandle->MT5CP2();
+      stdrec.hdr.gcp5iFix  = spillHandle->MT6CP2();
+      stdrec.hdr.tbccdi    = spillHandle->TBCCDI();
+      stdrec.hdr.tbccdo    = spillHandle->TBCCDO();
+      stdrec.hdr.mt6ca1    = spillHandle->MT6CA1();
+      stdrec.hdr.mt6ca2    = spillHandle->MT6CA2();
+      stdrec.hdr.mt6ca3    = spillHandle->MT6CA3();
+    }
+    catch(...) {
+      std::cout << "No spill info object found!  That's ok if this is MC" 
```

Comment:
assume this is to be addressed later what to do if it is MC or data because it should die if no spill info for data presumably. either way, i assume this is a TODO for later.